src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/impl/RmFieldRepublishService.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ons.census.fwmt.outcomeservice.service.impl;
```

Comment:
Move this to 
uk.gov.ons.census.fwmt.outcomeservice.message
and 
rename it to match the other Classes in this file RmFieldRepublishProducer (consistency)

Reply:
Done.